main/apps.py

```diff
@@ -1,26 +1,18 @@
 """
 Django app
 """
-from django.apps import AppConfig
-from django.conf import settings
-from django.core.exceptions import ImproperlyConfigured
+from typing import List
 
+from mitol.common.apps import BaseApp
 
-class RootConfig(AppConfig):
+
+class RootConfig(BaseApp):
     """AppConfig for this project"""
 
     name = "main"
 
-    def ready(self):
-        missing_settings = [
-            setting_name
-            for setting_name in settings.MANDATORY_SETTINGS
-            if getattr(settings, setting_name, None) in (None, "")
-        ]
-
-        if missing_settings:
-            raise ImproperlyConfigured(
-                "The following settings are missing: {}".format(
-                    ", ".join(missing_settings)
-                )
-            )
+    required_settings: List[str] = [
+        "MAILGUN_KEY",
+        "MAILGUN_SENDER_DOMAIN",
+        "SECRET_KEY",
+    ]
```

Comment:
I removed MAILGUN_* from my .env and no warning/errors showed up, should they?

Reply:
They should but the don't appear to be - this also seems to duplicate the behavior of `envs.validate()`, so i just removed this and put in a call that that instead. I'll need to figure out later why I seemingly built in 2 similar mechanisms into that library...